plugins/search-backend-module-explore/src/collators/ToolDocumentCollatorFactory.ts

```diff
@@ -87,7 +93,17 @@ export class ToolDocumentCollatorFactory implements DocumentCollatorFactory {
 
   private async fetchTools() {
     const baseUrl = await this.discovery.getBaseUrl('explore');
-    const response = await fetch(`${baseUrl}/tools`);
+
+    let headers = {};
+
+    if (this.tokenManager) {
+      const { token } = await this.tokenManager.getToken();
+      headers = {
+        Authorization: `Bearer ${token}`,
```

Comment:
Would it be possible to add this to the experimental backend module as well?
https://github.com/backstage/backstage/blob/master/plugins/search-backend-module-explore/src/collators/ToolDocumentCollatorFactory.ts#L89-L90

Reply:
Hey @camilaibs, the file you linked is the one I updated in this PR, right? Or am I missing anything? I was looking for where to change it, but it looks like it's the same Collator under the hood.